circuitbreaker.go

```diff
@@ -190,10 +208,11 @@ func NewRateBreaker(rate float64, minSamples int64) *Breaker {
 
 // Subscribe returns a channel of BreakerEvents. Whenever the breaker changes state,
 // the state will be sent over the channel. See BreakerEvent for the types of events.
+// Note that events may be dropped or not sent so clients should not rely on
```

Comment:
I think these comments accidentally got copied over from your other PR (ditto on the `AddListener` comment).

Reply:
Done.